cmake/CMakeLists_posix.txt

```diff
@@ -160,6 +177,34 @@ endforeach()
 
 install(TARGETS px4 DESTINATION bin)
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/ROMFS DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
-install(DIRECTORY ${PROJECT_SOURCE_DIR}/posix-configs DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
+install(DIRECTORY ${PROJECT_SOURCE_DIR}/posix-configs/SITL/init DESTINATION ${CMAKE_BINARY_DIR}/rootfs/etc/init)
+
+add_custom_command(TARGET px4
+	POST_BUILD
+	COMMAND mkdir -p ${CMAKE_BINARY_DIR}/rootfs/etc/
+	)
+
+add_custom_command(TARGET px4
+	POST_BUILD
+	COMMAND mkdir -p ${CMAKE_BINARY_DIR}/rootfs/etc/
+	)
+
+add_custom_command(TARGET px4
+	POST_BUILD
+	COMMAND cp -R ${PROJECT_SOURCE_DIR}/posix-configs/SITL/init ${CMAKE_BINARY_DIR}/rootfs/etc/
```

Comment:
We shouldn't use the POST_BUILD for these. It does not handle dependencies & changes to the script files. We should explicitly enumerate these files and use the `add_custom_command(OUTPUT` command, similar to the .msg files.

Then instead of `cp`, better would be `COMMAND ${CMAKE_COMMAND} -E copy`